src/reanimated2/screenTransition/styleUpdater.ts

```diff
@@ -0,0 +1,54 @@
+'use strict';
+import { isFabric } from '../PlatformChecker';
+import updateProps from '../UpdateProps';
+import type { ShadowNodeWrapper, SharedValue } from '../commonTypes';
+import type { Descriptor } from '../hook/commonTypes';
+import type {
+  PanGestureHandlerEventPayload,
+  ScreenTransitionConfig,
+} from './commonTypes';
+
+const IS_FABRIC = isFabric();
+
+const createViewDescriptor = IS_FABRIC
+  ? (screenId: number | ShadowNodeWrapper) => {
+      'worklet';
+      return { shadowNodeWrapper: screenId };
+    }
+  : (screenId: number | ShadowNodeWrapper) => {
+      'worklet';
+      return { tag: screenId, name: 'RCTView' };
+    };
```

Comment:
Some time ago we refactored our code to use the following pattern:
```suggestion
function createViewDescriptorPaper() {
  'worklet';
  return { tag: screenId, name: 'RCTView' };
}

function createViewDescriptorFabric() {
  'worklet';
  return { shadowNodeWrapper: screenId };
}

const createViewDescriptor = IS_FABRIC
  ? createViewDescriptorFabric
  : createViewDescriptorPaper;
```